logstash-output-elasticsearch.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-elasticsearch'
-  s.version         = '2.5.4'
+  s.version         = '3.0.0'
```

Comment:
Lets actually make it 5.0.0 so we immediately know what we're dealing with when a user reports an issue. normally, we want plugins to have their own version life cycle and follow semver, but this may be a special case where it warrants to track ES 5.0. Otherwise it may lead to a lot of confusing users and us